docs/Theming.md

```diff
@@ -22,28 +22,34 @@ The following colors can be configured:
 | `warning`               | Snotify warning popups, warning texts |
 | `error`                 | Snotify error popups, error texts |
 
-## Logos and Icons
-It is also possible to exchange the Dashboard logo and icons. You can replace the [assets](https://github.com/gardener/dashboard/tree/master/frontend/public/static/assets) folder via a volume mount in your deployment.
+If you use the helm chart, you can configure those with `frontendConfig.themes.light` for the light theme and `frontendConfig.themes.dark` for the dark theme.
 
-The simplest solution is to create a configmap and add the assets to it:
-```
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: dashboard-assets
-data:
-  logo.svg: |
-    ...
-```
+## Logos and Icons
 
-then you can map the assets to your dashboard in your deployment like this:
-```
-...
-volumes:
-  - configMap:
-    items:
-    name: dashboard-assets
-  name: assets
+It is also possible to exchange the Dashboard logo and icons. You can replace the [assets](https://github.com/gardener/dashboard/tree/master/frontend/public/static/assets) folder when using the [helm chart](https://github.com/gardener/dashboard/blob/master/charts/gardener-dashboard) in the `frontendConfig.assets` map.
+
+Attention: You need to set values for all files as mapping the volume will overwrite all files. It is not possible to exchange single files.
+
+The files have to be encoded as base64 for the chart - to generate the encoded files for the `values.yaml` of the helm chart, you can use this shorthand with `bash` or `zsh` on Linux systems:
+
+```bash
+cat << EOF
+  ###
+  ### COPY EVERYTHING BELOW THIS LINE
+  ###
+
+  assets:
+    favicon-16x16.png: |
+      $(cat frontend/public/static/assets/favicon-16x16.png | base64 -w0)
```

Comment:
-w0 does not work on macOS. Actually, it is the default mode in case of macOS. We should document this here

Reply:
For me it works because I am `base64` from [`coreutils`](https://formulae.brew.sh/formula/coreutils) on macOS
```bash
$ brew install coreutils
```

Reply:
In any case it should be documented how it can be run on MacOS :) I would vote for recommending to install `coreutils`

Reply:
I agree with @petersutter, will add this tomorrow.

Reply:
I added 

> If you use macOS, install coreutils with brew (`brew install coreutils`) or remove the `-w0` parameter.